index.html

```diff
@@ -11,21 +11,121 @@
+              <h1 class="display-4">Perfectly Imperfect  </h>
+              <p class="lead">Beauty is in the eye of the beholder</p>
+              <hr class="col-sm-4">
+              <p>I make and create all type of things. clothing and custom furniture.</p>
+              <p class="lead">
+              <a class="btn btn-primary btn-lg" href="#" role="button">Subscribe to My Page</a>
+              </p>
+          </div>
+          </div>
+      </div>
+      <div class="row">
+          <div class="col-sm-6">
+          <h2>Listen Up...</h2>
+          <p>Maecenas tincidunt semper tortor, at efficitur urna elementum in. In hac habitasse platea dictumst. Curabitur ornare dolor eu sem interdum posuere. Etiam vehicula sem ut nisi vulputate, eu feugiat diam facilisis. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Sed nec tempor nisl. Suspendisse et tristique ligula, non commodo ex. Praesent a vulputate nibh. Nulla ligula diam, molestie sed vestibulum sit amet, consectetur nec nulla. Etiam nec egestas sapien, ac volutpat mi.</p>
+          </div>
+          <div class="col-sm-6">
+              <h4>Stuff I'm Good At:</h4>
+              <ul class="list-group"></ul>
+              
+              <img src= "img/furniture.jpg" class="img-fluid" alt="Responsive image" height="60" width="60">
+              <li class="list-group-item">Furniture</li>
+              <img src= "img/owl.jpg" class="img-fluid" alt="Responsive image" height="42" width="42">
+              <li class="list-group-item">Clothing</li>
+              <img src= "img/clothing.jpg" class="img-fluid" alt="Responsive image" height="60" width="60">
+             <li class="list-group-item">Websites</li>
+              <img src= "img/anything.jpg" class="img-fluid" alt="Responsive image" media= height="42" width="42">
+
+              <li class="list-group-item">Jewlery</li>
+
+
+              <li class="list-group-item">Making anything my imagation can handle</li>
+          </ul>
+          </div>
+          <div class="card" style="width: 18rem;">
+            <img class="card-img-top" src="img/owl.jpg" alt="Card image cap">
+            <div class="card-body">
+              <h5 class="card-title">Craft me some Crafts</h5>
+              <p class="card-text">Place to craft and find your creative sidecommit.</p>
+              <a href="#" class="btn btn-primary">Go somewhere</a>
+            </div>
+          </div>
+      </div>
+      <div class="btn-group" role="group" aria-label="Basic example">
+        <button type="button" class="btn btn-secondary">like</button>
+        <button type="button" class="btn btn-secondary">tweeter</button>
+        <button type="button" class="btn btn-secondary">facebook</button>
+      </div>
+      <div class="row table-row no-gutters">
+          <h4>Work</h4>
+          <table class="table table-striped">
+          <thead>
+              <tr>
+              <th scope="col">Projects</th>
```

Comment:
Because you used columns here and rows below, your table data ended up rotated 90 degrees from where I think you wanted it. Just wanted to make sure you noticed, so you don't end up confused/frustrated the next time you want to make a table!